store/src/java/com/zimbra/cs/mailbox/MailItem.java

```diff
@@ -1914,7 +1918,11 @@ protected void finishCreation(MailItem parent) throws ServiceException {
         if (isLeafNode()) {
             boolean isDeleted = isTagged(Flag.FlagInfo.DELETED);
 
-            mMailbox.updateSize(mData.size, isQuotaCheckRequired());
+            if (trashFolderId == 3) {
```

Comment:
use folder constants for comparison refer https://github.com/Zimbra/zm-mailbox/blob/develop/common/src/java/com/zimbra/common/mailbox/FolderConstants.java#L27